### ISSUE TITLE
Error: 400 Bad Request when running `quiz.Schema.from_url`

### ISSUE THREAD

**Issue by rmarren1** [ariebovenberg/quiz#7]
I'm trying to use the auto-generate functionality on a graphql API I have running locally.

I get the following error:
```
>>> quiz.Schema.from_url('http://localhost:4000/graphql')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/site-packages/quiz/schema.py", line 350, in from_url
    result = execute(INTROSPECTION_QUERY, url=url, **kwargs)
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/site-packages/quiz/execution.py", line 77, in execute
    return snug.execute(snug_query, **kwargs)
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/site-packages/snug/query.py", line 218, in execute
    return exec_fn(query, client, _make_auth(auth))
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/site-packages/snug/query.py", line 141, in __execute__
    response = send(client, auth(request))
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/functools.py", line 820, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/home/ryan/.virtualenvs/graphql-client/lib/python3.7/site-packages/snug/clients.py", line 98, in _urllib_send
    res = opener.open(raw_req, **kwargs)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 400: Bad Request

```

On the graphql server, this error also appears:
```
message: "Cannot query field "onOperation" on type "__Directive"."
message: "Cannot query field "onFragment" on type "__Directive"."
message: "Cannot query field "onField" on type "__Directive"."
```

**Comment by ariebovenberg** [comment]
hi @rmarren1, thanks for posting your issue. 

It looks like something is wrong with the schema introspection query I have defined [here](https://github.com/ariebovenberg/quiz/blob/master/quiz/schema.py#L361).

It appears that inclusion of the `onOperation`, `onFragment` and `onField` fields is the problem. Fortunately, these fields should be able to be removed without affecting current functionality.

I should be able to release a patched version this week. If you're in a hurry, you can manually remove the offending lines [here](https://github.com/ariebovenberg/quiz/blob/master/quiz/schema.py#L376), and/or open a PR.

As an additional point, your case shows a that more descriptive error message may be helpful. This will be addressed in the future.

**Comment by ariebovenberg** [comment]
fixed as of release `0.0.4`. Closed.
